run_node.sh

```diff
@@ -26,7 +26,7 @@ Help()
 }
 
 echo "Updating this repo..."
-git pull origin main
```

Comment:
So right now user would need to run this script twice, to update this file. Maybe we should remove completely this step of updating repo and say to users to update it manually? I'm not sure whether our first idea of including this step in run_node script was correct